CefSharp.BrowserSubprocess/CefSharp.BrowserSubprocess.csproj

```diff
@@ -110,8 +110,11 @@
   <PropertyGroup>
     <PostBuildEvent>if $(PlatformName) == x86 (
   call "$(DevEnvDir)..\tools\vsvars32.bat"
-  editbin /largeaddressaware $(TargetPath)
-)</PostBuildEvent>
+  editbin /largeaddressaware $(TargetPath)  /TSAWARE $(TargetPath)
```

Comment:
do you need to specify `TargetPath` for each param? That seems like an unexpected behavior (I haven't looked at the doc).